arbos/block_processor.go

```diff
@@ -42,6 +44,8 @@ var EmitReedeemScheduledEvent func(*vm.EVM, uint64, uint64, [32]byte, [32]byte,
 var EmitTicketCreatedEvent func(*vm.EVM, [32]byte) error
 var gasUsedSinceStartupCounter = metrics.NewRegisteredCounter("arb/gas_used", nil)
 
+const NOT_EXPECTED_BUILDER_ERROR string = "This tx should not be executed by current builder"
```

Comment:
How about something like something like "This transaction is part of a block not built by the desired builder"?